Jenkinsfile

```diff
@@ -157,7 +163,7 @@ pipeline {
             script {
               def coverageEnabled = false
               def cmakeOptions = ""
-              if (!params.Linux) {
+              if (!params.Linux && (env.BRANCH_NAME in ['develop', 'master'] || env.CHANGE_ID != null)) {
```

Comment:
also the statement `(env.BRANCH_NAME in ['develop', 'master'] || env.CHANGE_ID != null)` is duplicated maybe move it into own method?

Reply:
might be the point. will be easier to work with it later

Reply:
Ok, we have to review the scripts in several sprints and refactor it.